src/services/basicValidationMapping.js

```diff
@@ -1,11 +1,20 @@
-/** Mapping between basicValidationType and the basic validation login
+/**
```

Comment:
I guess all this will get updated when you merge the OTP support PR into this right?

Reply:
Yeah.